sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/src/AssetFileTypeNotSupportedException.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+
+namespace Azure.MixedReality.ObjectAnchors.Conversion
+{
+    /// <summary>
+    /// An Exception thrown during an attempt to provide an unsupported asset file type in an asset conversion operation
+    /// </summary>
+    public class AssetFileTypeNotSupportedException : Exception, ISerializable
+    {
+        /// <summary>
+        /// Creates an instance of the UnsupportedAssetFileTypeException
```

Comment:
nit: need to update the docs for new name. Might be better to use `<see cref="AssetFileTypeNotSupportedException"/>`

Reply:
Replaced hard strings with this cref